### PR TITLE
1385001, customize a view.

### PR DESCRIPTION
**Pull request by john-par**
Update for task to use .NET tutorial template.

### CODE REVIEW

**Comment by john-par** [comment]
@tdykstra this PR is ready for your review. Please let me know what you'd like changed.

File: aspnet/mvc/overview/getting-started/database-first-development/customizing-a-view.md [hunk 1]
```diff
@@ -1,37 +1,55 @@
...
 
-> Using MVC, Entity Framework, and ASP.NET Scaffolding, you can create a web application that provides an interface to an existing database. This tutorial series shows you how to automatically generate code that enables users to display, edit, create, and delete data that resides in a database table. The generated code corresponds to the columns in the database table.
-> 
-> This part of the series focuses on changing the automatically-generated views to enhance the presentation.
+# Tutorial: Customize view for EF Database First with ASP.NET MVC app
 
+Using MVC, Entity Framework, and ASP.NET Scaffolding, you can create a web application that provides an interface to an existing database. This tutorial series shows you how to automatically generate code that enables users to display, edit, create, and delete data that resides in a database table. The generated code corresponds to the columns in the database table.
 
-## Add enrolled courses to student details
+This article focuses on changing the automatically-generated views to enhance the presentation.
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/database-first-development/customizing-a-view.md]: article --> tutorial

File: aspnet/mvc/overview/getting-started/database-first-development/customizing-a-view.md [hunk 2]
```diff
@@ -1,37 +1,55 @@
...
+Open **Views** > **Students** > *Details.cshtml*. Below the last &lt;/dl&gt; tag, but before the closing &lt;/div&gt; tag, add the following code.
 
 [!code-cshtml[Main](customizing-a-view/samples/sample1.cshtml)]
 
 This code creates a table that displays a row for each record in the Enrollment table for the selected student. The **Display** method renders HTML for the object (modelItem) that represents the expression. You use the Display method (rather than simply embedding the property value in the code) to make sure the value is formatted correctly based on its type and the template for that type. In this example, each expression returns a single property from the current record in the loop, and the values are primitive types which are rendered as text.
 
-Browse to the Students/Index view again and select **Details** for one of the students. You will see the enrolled courses have been included in the view.
+## Confirm courses are added
+
+Built the solution. Click **List of students** and select **Details** for one of the students. You will see the enrolled courses have been included in the view.
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/database-first-development/customizing-a-view.md]: Built --> "Build and run" --or just "Run"

File: aspnet/mvc/overview/getting-started/database-first-development/customizing-a-view.md [hunk 3]
```diff
@@ -1,37 +1,55 @@
...
-> [Previous](changing-the-database.md)
-> [Next](enhancing-data-validation.md)
+## Next steps
+In this tutorial, you:
+
+> [!div class="checklist"]
+> * Added courses to the student detail page
+> * Confirmed that the courses are added to the page
+
+Advance to the next article to learn how to add data annotations to the data model to specify validation requirements and display formatting.
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/database-first-development/customizing-a-view.md]: Advance to the next article to learn how to ~~add data annotations to the data model to~~ specify validation requirements and display formatting.

**Pull request merged**